03 State/src/nameEdit.jsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export const NameEditComponent = (props : { userName : string, onChange : (event : any) => any }) => {
```

Comment:
Same thing here, we are using types here, as far as I know ES6 is not typed, and I think we are not using other annotation, does this throws errors when transpiling (babel?)